tests/vmi_configuration_test.go

```diff
@@ -742,6 +742,21 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 			})
 		})
 
+		Context("[rfe_id:140][crit:medium][vendor:cnv-qe@redhat.com][level:component]with guest memory greater than request memory", func() {
+			It("[test_id:1669]should successfully start the VM when no VFIO device requested", func() {
```

Comment:
This is a new test. Is this test id correct? Same for the rfe_id

Reply:
I've updated this to remove the test id and rfe_id